eureka/S5_lightcurve_fitting/models/Model.py

```diff
@@ -298,9 +299,10 @@ def GPeval(self,  fit, **kwargs):
         # Get the time
         if self.time is None:
             self.time = kwargs.get('time')
-            
-        flux = np.ones(len(self.time)*self.nchan)
-        
+
+        # Set the default value
+        flux = np.zeros(len(self.time)*self.nchan)
```

Comment:
Reminder: Open question on what to do here.

Reply:
As decided over Zoom, we will leave the GP as additive for now and so this line is fine as is